test/ring/middleware/oauth2_test.clj

```diff
@@ -90,3 +90,93 @@
     (is (= {:status 200, :headers {}, :body tokens}
            (test-handler (-> (mock/request :get "/")
                              (assoc :session {::oauth2/access-tokens tokens})))))))
+
+
+
+
+; Tests with new parameter
+
+(def test-profile-2
```

Comment:
As far as I can tell, you've just duplicated the functions and tests. Ideally we want one extra test, and we want to check to make sure the basic authentication header is transferred.